ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/EthCall.java

```diff
@@ -127,8 +127,9 @@ private JsonCallParameter validateAndGetCallParams(final JsonRpcRequestContext r
       throw new InvalidJsonRpcParameters("Missing \"to\" field in call arguments");
     }
     if (callParams.getGasPrice() != null
-        && (callParams.getFeeCap().isPresent() || callParams.getGasPremium().isPresent())) {
-      throw new InvalidJsonRpcParameters("gasPrice cannot be used with baseFee or feeCap");
+        && (callParams.getMaxFeePerGas().isPresent()
+            || callParams.getMaxPriorityFeePerGas().isPresent())) {
+      throw new InvalidJsonRpcParameters("gasPrice cannot be used with baseFee or maxFeePerGas");
```

Comment:
I think it's an old issue but we need to replace baseFee in the message (maxPriorityFeePerGas ?)